.gitignore

```diff
@@ -41,3 +41,5 @@ doc/api/
 
 # End of https://www.toptal.com/developers/gitignore/api/dart,visualstudiocode
 /test/.gradle/
+
+*/.DS_Store
```

Comment:
Do not add it here. Please ignore it in your global `.gitignore`, but not in projects.